test/boostTest.cpp

```diff
@@ -28,24 +28,54 @@
 #define BOOST_TEST_NO_MAIN
 #include <boost/test/included/unit_test.hpp>
 #pragma GCC diagnostic pop
-
 #include <test/TestHelper.h>
+
 using namespace boost::unit_test;
 
+char** originalArgv;
+int originalArgc;
+static std::ostringstream strCout;
+std::streambuf* oldCoutStreamBuf;
+
 //Custom Boost Initialization
 test_suite* init_func( int argc, char* argv[] )
 {
 	if (argc == 0)
-		argv[1]=(char*)"a";
-
-	dev::test::Options::get();
+		argv[0] = (char*)"a";
```

Comment:
Boost requires this function to have   int argc, char* argv[]
and strict no warning requires each variable to be used.